tests/test_cedars_sinai.py

```diff
@@ -0,0 +1,93 @@
+from chargemaster_parsers.parsers.cedars_sinai import CedarsSinaiChargeMasterParser
+from chargemaster_parsers.parsers import ChargeMasterEntry
+
+import tempfile
+from openpyxl import Workbook
+import os
+import pprint
+import pytest
+import io
+
+@pytest.fixture
+def parser():
+    yield CedarsSinaiChargeMasterParser()
+
+def test_simple_row(parser):
+    wb = Workbook()
+    ws = wb.active
+
+    ws.cell(row=5, column=1, value="EAP PROC CODE")
+    ws.cell(row=5, column=2, value="EAP PROC NAME")
+    ws.cell(row=5, column=3, value="DEFAULT CPT/ HCPCS CODE")
+    ws.cell(row=5, column=4, value="DEFAULT OP FEE SCHEDULE")
+    ws.cell(row=5, column=5, value="IP/ED FEE SCHEDULE")
+
+    ws.cell(row=6, column=1, value=2600001)
+    ws.cell(row=6, column=2, value="HB IV INFUS HYDRATION 31-60 MIN")
+    ws.cell(row=6, column=3, value=96360)
+    ws.cell(row=6, column=4, value="$2161.71")
+    ws.cell(row=6, column=5, value="$2811.66")
+
+    ws.cell(row=7, column=1, value=2600017)
+    ws.cell(row=7, column=2, value="HB APPL ON-BODY INJECTOR SUBQ INJ")
+    ws.cell(row=7, column=3, value=96377)
+    ws.cell(row=7, column=4, value="$844.45")
+    ws.cell(row=7, column=5, value=None)
+
+    ws.cell(row=8, column=1, value=2700005)
+    ws.cell(row=8, column=2, value="HB CYTOTOXIC SPILL KIT")
+    ws.cell(row=8, column=3, value=None)
+    ws.cell(row=8, column=4, value="$254.09")
+    ws.cell(row=8, column=5, value=None)
+
+    expected_result = [
+        ChargeMasterEntry(
+            procedure_identifier = 2600001,
+            location = 'all',
+            procedure_description = "HB IV INFUS HYDRATION 31-60 MIN",
+            gross_charge = float(2161.71),
+            in_patient = False
+        ),
+        ChargeMasterEntry(
+            procedure_identifier = 2600001,
+            location = 'all',
+            procedure_description = "HB IV INFUS HYDRATION 31-60 MIN",
+            gross_charge = float(2811.66),
```

Comment:
Python will assume a float type from a decimal literal, but it doesn't hurt. See Line 64 it works the same.

Reply:
fixed